src/fs/list.js

```diff
@@ -1,5 +1,18 @@
+import { readdir } from 'fs/promises'
+import { throwError } from './error.js'
+
 const list = async () => {
-    // Write your code here 
+    try {
+        const PATH = './filess'
```

Comment:
double 's'! :-)

> const PATH = './file**SS**'

Reply:
it is for checking that throw Error is working